IdentityCore/src/pop_manager/MSIDDevicePopManager.m

```diff
@@ -69,13 +69,49 @@ - (MSIDAssymetricKeyPair *)keyPair
     return _keyPair;
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->